### PR TITLE
MVP release, version set to 0.4.0, fixes #1171

### LINKED ISSUE #1171: Upload pydeephaven to PYPI
For DHCE MVP, we want to publish the Python client to PYPI.

### PR DESCRIPTION
**Pull request by jmao-denver**

### CODE REVIEW

File: pyclient/setup.py [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
+    description='The Deephaven Python Client',
+    long_description=README,
+    long_description_content_type="text/markdown",
+    packages=find_packages(exclude=("tests",)),
     url='https://deephaven.io/',
     license='Deephaven Community License Agreement Version 1.0',
     author='Deephaven Data Labs',
-    author_email='jianfengmao@deephaven.io',
-    description='The Deephaven Python Client',
+    author_email='python@deephaven.io',
```
Comment (devinrsmith) [inline pyclient/setup.py:23]: Does this email address exist? I know we are/have created `developers@deephaven.io`.

File: pyclient/tests/test_console.py [hunk 2]
```diff
@@ -9,6 +10,7 @@
 
 class ConsoleTestCase(BaseTestCase):
 
+    @unittest.skip("GH ticket filed.")
```
Comment (devinrsmith) [inline pyclient/tests/test_console.py:13]: Please add a ticket number for any skipped cases.

File: pyclient/setup.py (hunk 1 above)
Comment (jmao-denver) [reply pyclient/setup.py:23]: it is used in the other two DH projects on PYPI

File: pyclient/tests/test_console.py (hunk 2 above)
Comment (jmao-denver) [reply pyclient/tests/test_console.py:13]: will do

**Review by devinrsmith — APPROVED** [review]

**Pull request merged**